scss/_patterns_cta.scss

```
@@ -0,0 +1,20 @@
+/*
+  @classreference
+  cta-block:
+    CTA Block:
+      .p-cta-block:
+        Main element of the CTA block.
+*/
+
+@import 'settings';
+
+@mixin vf-p-cta {
```

A bit of a nit, but let's use a full component name here for consistency:
```suggestion
@mixin vf-p-cta-block {
```